### PR TITLE
Update repository

### PR DESCRIPTION
**Pull request by dz333n**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2007 Weifen Luo (email: weifenluo@yahoo.com)
```
Comment (HBelusca) [inline LICENSE]: OK only the WinFormsUI is (C) Weifen Luo.
But the whole rest of RosDBG, according to the list of commits, should be (C) 2008-2011 Art Yerkes (@prozacchiwawa), (C) 2008-2011 Christopher von Wittich (@christophvw), (C) 2008-2011 Ged Murphy (@gedmurphy), (C) 2008-2011 Gregor Schneider (@grschneider), and other contributors.
Please check with the authors on Mattermost for example.

**Review by HBelusca — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (dz333n) [reply LICENSE]: Yeah, my mistake

Comment (dz333n) [reply LICENSE]: https://github.com/reactos/reactosdbg/pull/2

**Comment by dz333n** [comment]
Ready for merge!

**Review by HBelusca — APPROVED** [review]

**Pull request merged**